tests/retrieval/test_metrics.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
```

Comment:
general to the file, lest keep test file per metric unless there is good reason to merge it in one, also "test_metrics" is very bad names assuming that the packages is all about metrics

Reply:
Files separated, common logic now in the `helper`.